.github/workflows/db-migrate.yml

```
@@ -1,35 +1,38 @@
-        env:
-          VERCEL_ENV_TOKEN: ${{ secrets.VERCEL_ENV_TOKEN }}
-        run: npx vercel link -t $VERCEL_ENV_TOKEN --project=wahlen-foundation --yes
-
-      - name: Pull Vercel environment variables ▲
-        env:
-          VERCEL_ENV_TOKEN: ${{ secrets.VERCEL_ENV_TOKEN }}
-        run: npx vercel env pull .env -t $VERCEL_ENV_TOKEN --environment=production --yes
-
-      - name: Migrate the Database 🗃️
-        run: npm run db:migrate
+name: Migrate Database 🗃️
```

YAMLlint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)